app/javascript/packs/analytics.js

```diff
@@ -0,0 +1,3 @@
+import { descriptionSummary, descriptionToggleEvent } from 'analytics/school_description_helper.js';
```

Comment:
Why do we need a separate pack? 

Why not include in the main pack - we've not got HTTP2 turned on yet so will pay the roundtrip cost for whats is a very minimal amount of JS

I'd suggest instead moving the addition of the event listener to a function, and then including javascript tag at the foot of the html partial to call the code, instead of running code directly from the pack?